README.md

```diff
@@ -118,5 +118,10 @@ Some of the topics covered are:
 
 ### Sample projects
 * [Android App](sample/app) - Android use cases
+    - [auth](sample/app/src/main/kotlin/dev/gumil/kaskade/sample/auth) - Rx use case
```

Comment:
Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/manual/gumil/Kaskade/pullRequest?prid=5764048)